cordformation/src/main/kotlin/net/corda/plugins/Baseform.kt

```diff
@@ -283,13 +283,8 @@ open class Baseform(objects: ObjectFactory) : DefaultTask() {
 
     private fun invokeBootstrap(networkBootstrapperClass: Class<*>, rootDir: Path, allCordapps: List<Path>) {
         try {
-            if (networkParameterOverrides.packageOwnership.isEmpty()) {
-                val bootstrapMethod = networkBootstrapperClass.getMethod("bootstrapCordform", Path::class.java, List::class.java).apply { isAccessible = true }
-                bootstrapMethod.invoke(networkBootstrapperClass.newInstance(), rootDir, allCordapps)
-            } else {
-                val bootstrapMethod = networkBootstrapperClass.getMethod("bootstrapCordform", Path::class.java, List::class.java, String::class.java).apply { isAccessible = true }
-                bootstrapMethod.invoke(networkBootstrapperClass.newInstance(), rootDir, allCordapps, networkParameterOverrides.toConfig().root().render(ConfigRenderOptions.concise()))
-            }
+            val bootstrapMethod = networkBootstrapperClass.getMethod("bootstrapCordform", Path::class.java, List::class.java, String::class.java).apply { isAccessible = true }
+            bootstrapMethod.invoke(networkBootstrapperClass.newInstance(), rootDir, allCordapps, networkParameterOverrides.toConfig().root().render(ConfigRenderOptions.concise()))
```

Comment:
The Network Bootstrapper entry point supporting `networkParameterOverrides` was only added in Corda 4.3. Why have you removed support for the previous entry point? This breaks compatibility with earlier versions of Corda.

Reply:
ok, I didn't know that. That's probably what broke the unit tests. The reason I removed it it because it forces to write boilerplate code like 
```
if this is empty and that is null and that_other is null ...
then do this
```
but if it is necessary I'll do that